src/framework/components/anim/component.js

```diff
@@ -57,7 +57,7 @@ class AnimComponent extends Component {
         }
 
         // remove event from previous asset
-        if (this._stateGraphAsset) this.system.app.assets.get(this._stateGraphAsset).off('change', this._onStateGraphAssetChangeEvent);
+        if (this._stateGraphAsset) this.system.app.assets.get(this._stateGraphAsset).off('change', this._onStateGraphAssetChangeEvent, this);
```

Comment:
Would you mind splitting this into 2 lines please (get asset and then do `off`)?

Reply:
👍 Done